internal/adapter/http/v1/handler.go

```diff
@@ -2,13 +2,12 @@ package v1
 
 import (
 	"github.com/gin-gonic/gin"
```

Comment:
Here is the main achievement: no more dependency on the `service` package here and in the root handler as well. Nice!

Reply:
👏